spec/services/bulk_upload/lettings/year2022/row_parser_spec.rb

```diff
@@ -840,16 +840,6 @@
       end
     end
 
-    describe "#field_134" do
```

Comment:
why are we getting rid of these tests?

Reply:
previously for bulk upload, if there was an invalid radio option, we were adding a validation for that.
now that there are radio options different between years, we really kind of need to clear invalid answers unless we go down a fiddly rabbit hole where we try to check what year we are going from and to and selectively clear individual values.
 Phil is now going to do something else with bulk upload where he checks for fields that have been cleared in the saving process. That's probably more robust as well sense there are already values being cleared under various conditions.

Reply:
Sounds sensible!